wgpu-core/src/device/resource.rs

```diff
@@ -135,6 +138,9 @@ pub struct Device<A: HalApi> {
     pub(crate) deferred_destroy: Mutex<Vec<DeferredDestroy<A>>>,
     #[cfg(feature = "trace")]
     pub(crate) trace: Mutex<Option<trace::Trace>>,
+
+    // usage_scope pool
```

Comment:
that's not a useful comment, just repeats what's already in there in code. Either it explains again what this is for or we leave it since the type already does the main lifting of explaining